### PR TITLE
Update development version to 1.4.1 and release note format

### PR DESCRIPTION
**Pull request by ko1in1u**
This change is [Reviewable](https://reviewable.io/reviews/google/mobly-snippet-lib/128)

### CODE REVIEW

**Comment by xpconanfan** [comment]
You can't update the notes after the tag is pushed.
Let's update this the next release.

Can update the github one directly

**Comment by ko1in1u** [comment]
> > _[Reviewable](https://reviewable.io/reviews/google/mobly-snippet-lib/128)_ status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @ko1in1u and @xpconanfan)
> 
> _[`CHANGELOG` line 5 at r1](https://reviewable.io/reviews/google/mobly-snippet-lib/128#-NV_Sqd46aBnAQKX_rIH:-NV_Sqd46aBnAQKX_rII:b-b86mlm) ([raw file](https://github.com/google/mobly-snippet-lib/blob/9c9fa81ff357695d3cd9c15cd75b7f18e60a92fb/CHANGELOG#L5)):_
> 
> > ```
> > ## 1.4.0: Support RpcDefault and RpcOptional
> > 
> > With new annotations, the arguments in RPC can have default value or optional.
> > ```
> 
> nit: a default or optional value
> 
> _Code quote:_
> 
> ```
> default value or optional
> ```

Done

**Comment by xpconanfan** [comment]
No, pls don't re-push the tag!

**Review by xianyuanjia — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/mobly-snippet-lib/128)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @ko1in1u and @xpconanfan)

___
*[`CHANGELOG` line 5 at r1](https://reviewable.io/reviews/google/mobly-snippet-lib/128#-NV_Sqd46aBnAQKX_rIH:-NV_WnEIBGQF5TFA-K6t:b-4oeq2a) ([raw file](https://github.com/google/mobly-snippet-lib/blob/9c9fa81ff357695d3cd9c15cd75b7f18e60a92fb/CHANGELOG#L5)):*
<details><summary><i>Previously, ko1in1u (Kolin Lu) wrote…</i></summary><blockquote>

> > ```
> > ## 1.4.0: Support RpcDefault and RpcOptional
> > 
> > With new annotations, the arguments in RPC can have default value or optional.
> > ```
> > 

> 
> nit: a default or optional value
> 
> _Code quote:_
> 
> ```
> default value or optional
> ```
> 

Done


> You can't update the notes after the tag is pushed. Let's update this the next release.
> 
> Can update the github one directly
> 

Let me try if I can re-push the tag.

</blockquote></details>

Ref Ang's comment. Do not merge into this release.

**Review by xianyuanjia — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/mobly-snippet-lib/128)* status: 0 of 2 files reviewed, all discussions resolved (waiting on @xpconanfan)

___
*[`CHANGELOG` line 5 at r1](https://reviewable.io/reviews/google/mobly-snippet-lib/128#-NV_Sqd46aBnAQKX_rIH:-NVesAWS--rLLC_XfJ3G:bncdwh1) ([raw file](https://github.com/google/mobly-snippet-lib/blob/9c9fa81ff357695d3cd9c15cd75b7f18e60a92fb/CHANGELOG#L5)):*
<details><summary><i>Previously, xianyuanjia wrote…</i></summary><blockquote>

Ref Ang's comment. Do not merge into this release.
</blockquote></details>

1.4.0 released. Unblocking.

**Comment by ko1in1u** [comment]
> I don't think any code change is needed to "update dev version to 1.4.1"? So I'm not sure if that part of the title is needed

It merge to the same PR for those two files, should I deprecate this one and create another?

File: third_party/sl4a/gradle.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This version code implements the versioning recommendations in:
 # https://blog.jayway.com/2015/03/11/automatic-versioncode-generation-in-android-gradle/
-VERSION_CODE=1040099
-VERSION_NAME=1.4.0
+VERSION_CODE=1040100
+VERSION_NAME=1.4.1-SNAPSHOT
```
Comment (xpconanfan) [inline third_party/sl4a/gradle.properties:4]: we shouldn't change this now, should we?

**Review by xpconanfan — CHANGES_REQUESTED** [review]

File: third_party/sl4a/gradle.properties (hunk 1 above)
Comment (ko1in1u) [reply third_party/sl4a/gradle.properties:4]: Yeah, we should mark it to the next development iteration after released a new version.

**Pull request closed without merging**